mattermost/umbrel-app.yml

```diff
@@ -31,26 +31,8 @@ defaultUsername: ""
 defaultPassword: ""
 dependencies: []
 releaseNotes: >-
-  Mattermost v10.3.1 and v10.3.0 contain multiple new features, improvements, and bug fixes.
-
-
-  Key Updates:
-
-  - New draft messages view
-
-  - Copilot plugin enabled by default
-
-  - Updated minimum Edge and Chrome versions to 130+
-
-  - Classic Mobile App deprecated—upgrade to v2 Mobile App
-
-
-  Fixes:
-
-  - Improved Desktop App performance, post drafts, and status syncing
-
-  - Fixed user status syncing and search input accessibility issues
+  Mattermost Platform Release 10.4.1 includes various new improvements and bug fixes.
 
 
   Full release notes are available at https://docs.mattermost.com/about/mattermost-v10-changelog.html
-path: ""
+path: ""
```

Comment:
@nmfretz final newline missing here.

Reply:
Thanks @al-lac. Added back in here: https://github.com/getumbrel/umbrel-apps/pull/2080/commits/0ba25d70ffe6bde3cb0ca0b8729b7a112d608064. The updater ends up removing newlines at the end of the yml files. Is there a reason we should be keeping them?


I've also added release notes from the changelog https://github.com/getumbrel/umbrel-apps/pull/2080/commits/3966429efb58734a6ce57cf4f005eca4966dcfe2.

Reply:
We should keep them because of the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206). Also GitHub shows when new lines are missing at the end.